audit/projects/k8s-infra-prow-build/services/monitoring/dashboards/prow-build-ephemeral-ssd-experiment.json

```diff
@@ -0,0 +1 @@
s":{"label":"y1Axis","scale":"LINEAR"}}},{"title":"pool5 throttled write bytes","xyChart":{"chartOptions":{"mode":"COLOR"},"dataSets":[{"minAlignmentPeriod":"60s","plotType":"LINE","targetAxis":"Y1","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"compute.googleapis.com/instance/disk/throttled_write_bytes_count\" resource.type=\"gce_instance\" metadata.system_labels.\"instance_group\"=monitoring.regex.full_match(\"gke-prow-build-pool5-.*\")","secondaryAggregation":{"alignmentPeriod":"60s","perSeriesAligner":"ALIGN_MEAN"}}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}},{"title":"pool4 throttled read bytes","xyChart":{"chartOptions":{"mode":"COLOR"},"dataSets":[{"minAlignmentPeriod":"60s","plotType":"LINE","targetAxis":"Y1","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"compute.googleapis.com/instance/disk/throttled_read_bytes_count\" resource.type=\"gce_instance\" metadata.system_labels.\"instance_group\"=monitoring.regex.full_match(\"gke-prow-build-pool4-.*\")","secondaryAggregation":{"alignmentPeriod":"60s","perSeriesAligner":"ALIGN_MEAN"}}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}},{"title":"pool5 throttled read bytes","xyChart":{"chartOptions":{"mode":"COLOR"},"dataSets":[{"minAlignmentPeriod":"60s","plotType":"LINE","targetAxis":"Y1","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"compute.googleapis.com/instance/disk/throttled_read_bytes_count\" resource.type=\"gce_instance\" metadata.system_labels.\"instance_group\"=monitoring.regex.full_match(\"gke-prow-build-pool5-.*\")","secondaryAggregation":{"alignmentPeriod":"60s","perSeriesAligner":"ALIGN_MEAN"}}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}},{"text":{"content":"Comparing the throttled read/write bytes on a per-instance basis for:\n\n- pool4 - 500GB pd-ssd ephemeral storage\n- pool5 - 2 x 375 GB local-ssd ephemeral storage","format":"MARKDOWN"},"title":"What this Dashboard is For"},{"title":"Kubernetes Pod - Volume usage (filtered) [MAX]","xyChart":{"chartOptions":{"mode":"COLOR"},"dataSets":[{"minAlignmentPeriod":"60s","plotType":"LINE","targetAxis":"Y1","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","crossSeriesReducer":"REDUCE_MAX","groupByFields":["metadata.user_labels.\"prow.k8s.io/job\""],"perSeriesAligner":"ALIGN_MAX"},"filter":"metric.type=\"kubernetes.io/pod/volume/used_bytes\" resource.type=\"k8s_pod\" resource.label.\"namespace_name\"!=\"default\" metadata.user_labels.\"prow.k8s.io/type\"=\"periodic\"","secondaryAggregation":{"alignmentPeriod":"60s"}}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}}]},"name":"projects/773781448124/dashboards/f0163540-a8b7-4618-8308-66652d3d4794"}
```

Comment:
This was me building the dashboard mentioned in https://github.com/kubernetes/k8s.io/issues/1187#issuecomment-928446704